data_model/src/transaction.rs

```diff
@@ -79,16 +79,34 @@ pub trait Txn {
 }
 
 /// Either ISI or Wasm binary
-#[derive(
-    Debug, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, FromVariant, IntoSchema,
-)]
+#[derive(Debug, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema)]
 pub enum Executable {
     /// Ordered set of instructions.
     Instructions(Vec<Instruction>),
     /// WebAssembly smartcontract
     Wasm(Vec<u8>),
```

Comment:
shouldn't this be
```suggestion
    Wasm(WasmSmartContract),
```

Reply:
otherwise I don't see `WasmSmartContract` used anywhere else

Reply:
Rebase artifact.